test/testsimplifytypedef.cpp

```diff
@@ -3352,6 +3352,19 @@ class TestSimplifyTypedef : public TestFixture {
                "    a** p = new a * [N];\n"
                "}\n";
         TODO_ASSERT_EQUALS("does not compile", "void f ( int N ) { int ( * * p ) [ 7 ] ; p = new int ( * ) [ N ] [ 7 ] ; }", tok(code));
+
+        // #11772
+        code = "typedef t;\n" // don't crash on implicit int
+               "void g() {\n"
+               "    sizeof(t);\n"
+               "}\n";
+        ASSERT_EQUALS("void g ( ) { sizeof ( t ) ; }", tok(code));
```

Comment:
I didn't know that there could be implict int in typedefs. You taught me one more detail today. :-)

Could you write a TODO comment. I have the feeling that the t should be replaced by "int" in the output code.

Reply:
I only learned about it from daca. That code is accepted by MSVC, gcc (with warning), but rejected by clang.